tools/roslaunch/src/roslaunch/xmlloader.py

```diff
@@ -216,6 +223,7 @@ def reqd_attrs(self, tag, context, attrs):
 
     def _check_attrs(self, tag, context, ros_config, attrs):
         tag_attrs = tag.attributes.keys()
+        
```

Comment:
Unrelated change. Should be be reverted.

Reply:
Reverted.